SPECS/moby-containerd/moby-containerd.spec

```diff
@@ -17,6 +17,7 @@ Source1: containerd.service
 Source2: containerd.toml
 Source3: NOTICE
 Source4: LICENSE
+Patch0:  1.4-Use-chmod-path-for-checking-symlink.patch
```

Comment:
IIRC the patch file name needs to be `CVEnumber.patch` so our CVE tooling will auto-detect the CVE is handled.

Reply:
Done